### PR TITLE
Implemented the video-player for android

### PR DESCRIPTION
**Pull request by emadbaqeri**
The Video Player feature has been implemented but is not working on iOS yet.
      Note:
        - Based on the needs and package compatibility the gradle version has
        been updated to `gradle-7.5-all`
        - `jcenter` has been added to the `build.gradle` for installing the
          dependencies of `expo-av`

### CODE REVIEW

File: app/screens/image-gallery-viewer/gallery-image.tsx [hunk 1]
```diff
@@ -43,13 +47,31 @@ const MAX_SCALE = 6
...
+  const { assets } = await MediaLibrary.getAssetsAsync({
+    mediaType: ["photo", "video", "audio", "unknown"],
+  })
+  if (!assets) {
+    return null
+  }
+  return assets
+}
+
+const getMediaInfo = async (asset: Asset): Promise<MediaLibrary.AssetInfo> => {
```
Comment (ghorbani-m) [inline app/screens/image-gallery-viewer/gallery-image.tsx:63]: @emadbaqeri Can you please define this function (getMediaInfo) in the asset-service.ts?

File: app/screens/image-gallery-viewer/gallery-image.tsx [hunk 2]
```diff
@@ -43,13 +47,31 @@ const MAX_SCALE = 6
 const SWIPE_UP_THRESHOLD = 10
 const SWIPE_TO_CLOSE_THRESHOLD = 100
 
+import * as MediaLibrary from "expo-media-library"
+import type { Asset as Media } from "expo-media-library"
+
+const getUserAssets = async (): Promise<Media[] | null> => {
```
Comment (ghorbani-m) [inline app/screens/image-gallery-viewer/gallery-image.tsx:53]: @emadbaqeri  We have already defined this function on the asset-service.ts and you can use it

File: ios/fotos/Info.plist [hunk 3]
```diff
@@ -35,6 +35,8 @@
 			</dict>
 		</dict>
 	</dict>
+	<key>NSMicrophoneUsageDescription</key>
```
Comment (ghorbani-m) [inline ios/fotos/Info.plist:38]: @emadbaqeri  Is it mandatory to get Microphone permission to play a video?

Comment (emadbaqeri) [reply ios/fotos/Info.plist:38]: Hi @ghorbani-m, according to [the documentation](https://github.com/expo/expo/tree/sdk-46/packages/expo-av#installation-in-bare-react-native-projects), yeah we have to do that.

File: app/screens/image-gallery-viewer/gallery-image.tsx (hunk 2 above)
Comment (emadbaqeri) [reply app/screens/image-gallery-viewer/gallery-image.tsx:53]: As you can see I'm not using that anywhere in the code I should have remove it ;)
Thanks for mentioning by the way

**Pull request merged**